AggregateMetrics/AggregateMetrics/AzureWebApp/Implementation/PerformanceCounterFromJsonGauge.cs

```diff
@@ -18,22 +18,29 @@ internal class PerformanceCounterFromJsonGauge : ICounterValue
         /// </summary>
         private string jsonId;
 
+        /// <summary>
+        /// Identifier of the environment variable.
+        /// </summary>
+        private AzureWebApEnvironmentVariables environmentVariable;
```

Comment:
ok... we will need to start using it at some point. I'd say it needs to be passed to cacheHelper so it knows which part of JSON to read or which environment variable's value to take